client/components/tinymce/plugins/embed/test/dialog.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import Dialog from 'components/dialog';
```

Comment:
Though I get that they are external to this 'unit' I think we should keep `import Dialog from 'components/dialog';` and `import FormTextInput from 'components/forms/form-text-input';` in the internal deps.

Reply:
Doh, I don't think I meant to put those in `External`; thanks for catching that :)

Reply:
NP! :)